samples/ios/app/metrics.yaml

```diff
@@ -28,6 +28,9 @@ browser.engagement:
       key2:
         type: string
         description: "This is key two"
+      And1WithUnusualCASING:
```

Comment:
Lots of repetition...  I wonder if we could make better use of anchors at some point to cut down on this 🤔  (a problem for the future, obviously)